server/server.go

```diff
@@ -400,6 +403,7 @@ func NewServer(configFile string) (*Server, error) {
 			SkipLogging: true,
 		}))
 	}
+	server.Healthcheck = *healthcheck.NewHealthcheck(server.db, server.sync)
```

Comment:
why not just keep it as a pointer in server struct?

Reply:
changed